test/metrix/adapters/log_test.exs

```diff
@@ -1,22 +1,27 @@
-defmodule ElixirTools.Metrix.Adapters.LogTest do
+defmodule Metrix.Adapters.LogTest do
   use ExUnit.Case, async: false
 
   import ExUnit.CaptureLog
+  import ElixirTools.MetrixHelper
 
+  alias ElixirTools.Metrix
   alias ElixirTools.Metrix.Adapters.Log
 
+  setup :start_supervisor
+
   setup do
     Logger.configure(level: :debug)
+    :ok
   end
 
   test "connect" do
-    log_message = capture_log(&Log.connect/0)
+    log_message = capture_log(fn -> Log.connect() end)
```

Comment:
What happens when we use the shorthand?

Reply:
it's the same, I just tested something inside and forgot to reverse it